docs/en/reference/second-level-cache.rst

```diff
@@ -203,11 +195,11 @@ Cache Factory is the main point of extension.
 
 It allows you to provide a specific implementation of the following components :
 
-* ``QueryCache`` Store and retrieve query cache results.
-* ``CachedEntityPersister`` Store and retrieve entity results.
-* ``CachedCollectionPersister`` Store and retrieve query results.
-* ``EntityHydrator``  Transform an entity into a cache entry and cache entry into entities
-* ``CollectionHydrator`` Transform a collection into a cache entry and cache entry into collection
+* ``QueryCache`` stores and retrieves query cache results.
+* ``CachedEntityPersister`` stores and retrieves entity results.
+* ``CachedCollectionPersister`` stores and retrieves query results.
+* ``EntityHydrator``  transforms an entity into a cache entry and caches entry into entities
+* ``CollectionHydrator`` transforms a collection into a caches entry and cache entry into collection
```

Comment:
This could be reverted and converted to a description list (see http://html5doctor.com/the-dl-element/)

```suggestion
``QueryCache``
    stores and retrieves query cache results.
``CachedEntityPersister``
    stores and retrieves entity results.
``CachedCollectionPersister``
    stores and retrieves query results.
``EntityHydrator``
    transforms an entity into a cache entry and caches entry into entities
``CollectionHydrator``
    transforms a collection into a caches entry and cache entry into collection
```